frontend/Views/Admin/Dashboard/actions.js

```diff
@@ -28,7 +28,7 @@ export const getAdminDashboardInfo = () => {
 
     getAdminDashboardInfoAPI().then(
```

Comment:
NOTE: The changes in this file was just a fix for some dead code I saw. However, I haven't tested it. This should probably be tested